index.bs

```diff
@@ -13446,6 +13446,12 @@ agents in referencing specifications be linked to the
 </blockquote>
 
 
+<h2 id="priv-sec">Privacy and Security Considerations</h2>
+
+This specification introduces no privacy or security considerations, as it is defining a
+meta-language that is used to write specifications that actually define features.
```

Comment:
I think we should say something about the nature of IDL. That it defines a security boundary of sorts between ECMAScript and IDL values and that not implementing the conversion layer correctly can lead to security issues. As well as not fully understanding the implications of object/any or [AllowShared].

Reply:
Agree, WebIDL is not privacy and security agnostic. It actually provides a bunch of useful invariants/behaviors at the binding layer.